CMakeLists.txt

```diff
@@ -1,4 +1,15 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
This update is because we are already using cmake_link_options in this file, which requires cmake 3.13. Nobody has complained so I figured it was safe.

Reply:
This breaks out current circleci build: https://app.circleci.com/pipelines/github/emscripten-core/emsdk/921/workflows/3e087c5c-71e9-409c-99c2-40d7495c084f/jobs/4118

We currently build using ubuntu/bionic (18.04) which has cmake version 3.10.2

Reply:
I think you meant `add_link_options` .. which is only used in the EMSCRIPTEN build so it not really hard dependency yet (and we should probably remove it in favor of `add_link_flag` for consistency).